tests/test-slurmdb.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import, unicode_literals
+
+import pyslurm
```

Comment:
I would reorder these as Python builtin modules, 3rd party modules and custom modules, all separated by a space.